dnf-plugins-core.spec

```diff
@@ -9,7 +10,8 @@
 %endif
 
 Name:           dnf-plugins-core
-Version:        1.0.0
+Epoch:          1
```

Comment:
but this is needed only for Obsoletes/Provides, not for dnf-plugins-core themselves